SPEC.md

```diff
@@ -101,4 +101,13 @@ The test index makes use of symbolic language in describing connection and messa
 
     <>
     -> ping message with random `sequence` number
-    <- pong response with the same `sequence` number
\ No newline at end of file
+    <- pong response with the same `sequence` number
+
+### ZG-CONFORMANCE-004
+    
+    The node responds with mtLEDGER_DATA for mtGET_LEDGER
```

Comment:
```suggestion
    The node responds with mtLEDGER_DATA for mtGET_LEDGER:
    
```

Reply:
this should probably be a dot, like in the earlier ones

Reply:
Suggestion updated.